ci/tasks/sonobuoy_e2e.py

```diff
@@ -30,43 +30,33 @@ def cleanup_cluster(self, sonobuoy_args):
         logger.info(delete)
 
     def collect_results(self, retries, sonobuoy_args):
-        error = None
         attempts = 0
 
-        if 'Sonobuoy has completed' in self._get_status():
-            results = None
-
-            while retries > attempts:
-                try:
-                    attempts += 1
-                    logger.info(f'Attempting to retrieve the results {attempts}/{retries}')
-                    results = self._sonobuoy('retrieve results' + ' '.join(sonobuoy_args))
-
-                except SonobuoyE2eTestsError:
-                    if retries == attempts:
-                        error = 'Could not retrieve sonobuoy results'
-                        break
-                    time.sleep(self.default_sleep)
-                else:
-                    break
+        results = None
 
-            self._extract_results(results.strip())
-        else:
-            error = 'Sonobuoy e2e tests failed'
+        while retries > attempts:
+            try:
+                attempts += 1
+                logger.info(f'Attempting to retrieve the results {attempts}/{retries}')
+                results = self._sonobuoy('retrieve results' + ' '.join(sonobuoy_args))
+
+            except SonobuoyE2eTestsError:
+                if retries == attempts:
+                    error = 'Could not retrieve sonobuoy results'
+                    break
+                time.sleep(self.default_sleep)
+            else:
+                break
 
-        if error is not None:
```

Comment:
why are you removing this check? Does it make sense to try to extract results if there was an error collecting them?

Reply:
Umm, the error part is not used anymore but I see that we are no longer raising an error if we could not retrieve the results.

let me rework this